src/components/SignUp/SignUpScreen.js

```diff
@@ -1,65 +1,282 @@
-import React, { useState, useCallback, useMemo } from 'react';
-import TextField from '@material-ui/core/TextField';
-import MenuItem from '@material-ui/core/MenuItem';
-import InputLabel from '@material-ui/core/InputLabel';
-import Select from '@material-ui/core/Select';
-import './SignUp.css';
-import Grid from '@material-ui/core/Grid';
-import Button from '@material-ui/core/Button';
+import React, { useMemo, useState, useCallback } from 'react';
```

Comment:
For your signup screen notice how the text field values bleed through the side navbar? (When your window is small and click the navbar extension icon in the upper right part of the screen). To fix that, you can change the z-index for text inputs to 0. Refer to my newly pushed SignInScreen if you need help with that!